simplivity/resources/certificates.py

```diff
@@ -51,6 +51,20 @@ def get_by_name(self, name):
         """ Method not available on resource"""
         raise exceptions.HPESimpliVityMethodNotSupportedError("Method get_by_name is not supported")
 
+    def add_certificate(self, certificate, timeout=-1):
+        """Add a SSL certificate to the HPE SimpliVity trust store
+
+        Args:
+            certificate: Base64 encoded SSL certificate.
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            dict : Returns the certificate details.
+        """
+
+        data = {"certificate": certificate}
+        return self._client.do_post(URL, data, timeout)
```

Comment:
I think, we should return the created certificate object.

Reply:
Hi Sijeesh,
POST command returns the certificate details

Reply:
Got it. I was thinking if it returns a single certificate object details we can create an object of the certificate and return it to match with the other resource implementation.